.gitlab/kernel_version_testing/system_probe.yml

```diff
@@ -159,4 +190,25 @@ kernel_matrix_testing_upload_deps:
       - $LibvirtSSHKeyARM.pub
       - $STACK_DIR
 
-
+kernel_matrix_testing_run_tests:
+  stage: kernel_matrix_testing
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/test-infra-definitions/runner:7c39524b
+  needs: ["kernel_matrix_testing_upload_deps", "kernel_matrix_testing_setup_env"]
+  tags: ["arch:amd64"]
+  variables:
+    AWS_EC2_SSH_KEY_FILE: $CI_PROJECT_DIR/ssh_key
+    GOVERSION: "1.19"
+  before_script:
+    - !reference [.kernel_matrix_testing_new_profile]
+  script:
+    - INSTANCE_IP=$(cat $CI_PROJECT_DIR/stack.outputs | grep $ARCH-instance-ip | cut -d ' ' -f 2)
+    - MICRO_VM_IP=$(cat $CI_PROJECT_DIR/stack.outputs | grep $ARCH-$TAG | cut -d ' ' -f 2)
+    - chmod 600 $AWS_EC2_SSH_KEY_FILE
+    # Without the newline ssh silently fails and moves on to try other auth methods
+    - echo "" >> $AWS_EC2_SSH_KEY_FILE
+    # ssh into each micro-vm and run initialization script. This script will also run the tests.
+    - ssh -o StrictHostKeyChecking=no -i $AWS_EC2_SSH_KEY_FILE ubuntu@$INSTANCE_IP "ssh -o StrictHostKeyChecking=no -i /home/kernel-version-testing/ddvm_rsa root@${MICRO_VM_IP} 'cp /opt/kernel-version-testing/dependencies-${ARCH}.tar.gz / && cd / && tar xzf dependencies-${ARCH}.tar.gz --strip-components=1 && ls -la / && chmod +x /micro-vm-init.sh && /micro-vm-init.sh ${GOVERSION}'"
```

Comment:
If the init commands get any longer, they will probably need to be in a script file.